src/lib/DateInput.svelte

```diff
@@ -182,6 +185,8 @@
     &:focus
       border-color: var(--date-picker-highlight-border, #0269f7)
       box-shadow: 0px 0px 0px 2px var(--date-picker-highlight-shadow, rgba(#0269f7, 0.4))
+    &:disabled
+      color: rgba(103, 113, 137, 0.3)
```

Comment:
```suggestion
      opacity: 0.5
```

Let's not use color because that won't work when people have custom color themes